packages/driver/test/cypress/integration/commands/assertions_spec.js

```diff
@@ -976,6 +976,110 @@ describe('src/cy/commands/assertions', () => {
     })
   })
 
+  context('format quotation marks', () => {
+    const expectMarkdown = (test, message, done) => {
+      cy.then(() => {
+        test()
+      })
+
+      cy.on('log:added', (attrs, log) => {
+        if (attrs.name === 'assert') {
+          cy.removeAllListeners('log:added')
+
+          expect(log.get('message')).to.eq(message)
+
+          done()
+        }
+      })
+    }
+
+    it('preserves quotation marks in number strings', (done) => {
+      expectMarkdown(() => {
+        try {
+          expect(25).to.eq('25')
+        } catch (error) {} /* eslint-disable-line no-empty */
+      },
+      `expected **25** to equal **'25'**`,
+      done)
+    })
+
+    it('preserves quotation marks in empty string', (done) => {
+      expectMarkdown(() => {
+        try {
+          expect(42).to.eq('')
+        } catch (error) {} /* eslint-disable-line no-empty */
+      },
+      `expected **42** to equal **''**`,
+      done)
+    })
+
+    it('preserves quotation marks if escaped', (done) => {
+      expectMarkdown(
+        () => expect(`\'cypress\'`).to.eq(`\'cypress\'`),
+        // ****'cypress'**** -> ** for emphasizing result string  + ** for emphasizing the entire result.
```

Comment:
thanks for adding this comment :pray:

Reply:
Actually, when I saw this, I was too wondering why this happened. 

And thought many developers who will read this code later would be confused when they saw `****`. That's why I've added this comment.